app/models/null_bid.rb

```diff
@@ -0,0 +1,7 @@
+class NullBid
+  def amount
```

Comment:
I know these return nil by default, but should we add a comment that this is happening here?